syntax = "proto3";
package dbclient;




import "database_delivery_sdk/model/database_delivery/dbclient.proto";

/*
ListClient请求
*/
message ListClientRequest {
    /*
    页码
    */
    int32 page = 1 ;
    /*
    每页数据量
    */
    int32 pageSize = 2 ;
}

/*
ListClient返回
*/
message ListClientResponse {
    /*
    页数
    */
    int32 page = 1 ;
    /*
    页大小
    */
    int32 page_size = 2 ;
    /*
    总数
    */
    int32 total = 3 ;
    /*
    数据列表
    */
    repeated database_delivery.DBClient list = 4 ;
}

/*
ListClientApi返回
*/
message ListClientResponseWrapper {
    /*
    返回码
    */
    int32 code = 1 ;
    /*
    返回码解释
    */
    string codeExplain = 2 ;
    /*
    错误详情
    */
    string error = 3 ;
    /*
    返回数据
    */
    dbclient.ListClientResponse data = 4 ;
}

